Makefile.core.mk

```diff
@@ -50,11 +54,24 @@ generate: generate-values generate-types vfsgen
 
 clean: clean-values clean-types
 
-default: mesh
+default: mesh controller
 
 mesh: vfsgen
 	go build -o ${GOBIN}/mesh ./cmd/mesh.go
 
+controller: vfsgen
+	go build -o ${CONTROLLER_BUILD}/_output/bin/istio-operator ./cmd/manager
+
+controller-image: build-controller-image push-controller-image
+
+build-controller-image: controller
+	docker build -t istionightly/operator:latest -f ${CONTROLLER_BUILD}/Dockerfile .
```

Comment:
please do not tag images as `:latest`.  You can use `:$(VERSION}`.

Reply:
switched to `:$(VERSION}`